domain/src/main/java/com/yapp/domain/formatter/AlarmDateTimeFormatter.kt

```
@@ -0,0 +1,184 @@
+                    formattedTimeOrDateTime = alarmOccurrenceDateTime.format(DateTimeFormatter.ofPattern(formats.todayTimePattern))
+                    return String.format(formats.tomorrow, formattedTimeOrDateTime)
+                }
+                // 4. 그 외의 경우 (년도가 같고, 오늘이나 내일이 아닌 다른 날) 'thisYear' 포맷 적용
+                else -> {
+                    formattedTimeOrDateTime = alarmOccurrenceDateTime.format(DateTimeFormatter.ofPattern(formats.thisYearDatePattern))
+                    return String.format(formats.thisYear, formattedTimeOrDateTime)
+                }
+            }
+        } catch (e: DateTimeParseException) {
+            formats.noAlarm
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)